server/src/main/java/org/spine3/server/ClientService.java

```diff
@@ -112,7 +112,7 @@ public void run() {
     }
 
     @VisibleForTesting
-    /* package */ boolean isShutdown() {
+    public boolean isShutdown() {
```

Comment:
Is needed in tests, which I moved to the separate package from the root one (because we have separate messages and factories for tests).